### PR TITLE
Modified Video Code,Portfolio Cards,Animation

### PR DESCRIPTION
**Pull request by Disha-Dutta**
- Fixed youtube video layout
- Created Cards for portfolio layout
- Added few new animations
- Components Refractor.

### CODE REVIEW

**Comment by Disha-Dutta** [comment]
deleme


deleme2


deletme3

File: _includes/css/agency.css [hunk 1]
```diff
@@ -1428,3 +1428,26 @@ div.testimonialcontent{
...
+
+
+/* video close button */
+.fancybox-button .fancybox-button--close{
+    background-color:#fed136;
+}
+
+.fancybox-button, .fancybox-button:link, .fancybox-button:visited{
+    color: #fff;
+}
```
Comment (sahilbathla) [inline _includes/css/agency.css:1453]: use new line

File: _includes/css/agency.css [hunk 2]
```diff
@@ -1428,3 +1428,26 @@ div.testimonialcontent{
...
+.card:hover{
+    box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
+}
+
+
+/* Project card css ends here */
+
+
+/* video close button */
+.fancybox-button .fancybox-button--close{
```
Comment (sahilbathla) [inline _includes/css/agency.css:1447]: space around { and other operators

File: _includes/header.html [hunk 3]
```diff
@@ -55,7 +61,12 @@
                 <div class="col-lg-12 landing-page-image-container">
                     <div class="col-lg-4 col-xs-12 landing-page-text">
                         <h1><span style="-webkit-text-stroke-width: 2px; -webkit-text-stroke-color: #66C7FE; color: transparent;">WE create</span> <br>awesome <br> WEB & MOBILE APPS</h1>
-                        <img id="btn-video" class="img-responsive video-play-button" src="img/play_button.svg" alt="">
+                        <div class="video">
+                                <a data-fancybox href="https://www.youtube.com/embed/wUvjxucMft8?autoplay=1&mute=1&enablejsapi=1">
```
Comment (sahilbathla) [inline _includes/header.html:65]: can we use 2 space indentation please

File: _includes/portfolio_grid.html [hunk 4]
```diff
@@ -34,9 +38,32 @@ <h4>{{ post.title }}</h4>
...
                     {% endfor %}
            
-            </div>
+            </div> -->
+<!-- Original code ends here -->
+            <div class="row">
+                    <div style="padding: 30px;" class="">
+                        <h3 class="text-center" style="color:black"><span style="color:#fed618">Live </span>Projects</h3>
+                    </div>
+                {% for post in site.posts %}
```
Comment (sahilbathla) [inline _includes/portfolio_grid.html:47]: indentation is off, it will complicate the code, can you fix it please?

**Review by sahilbathla — CHANGES_REQUESTED** [review]
1. Can you add screenshots for mobile too, as I said its mobile-first
2. Need `padding-bottom` in project tiles between technologies and end of the tile

**Pull request closed without merging**